share/commands/agent-run

```diff
@@ -70,6 +72,10 @@ while getopts "uiIvdqwrRmcb:D:f" opt; do
       QUIET=1
       UPDATE_OPTIONS="${UPDATE_OPTIONS} -d"
       ;;
+    g)
+      CLASSES="${CLASSES},force_full_compliance"
```

Comment:
I guess this will break (segfault?) if there are no classes previously defined. We could add multiple -D options here